trustfall_core/test_data/tests/valid_queries/fold_with_outputs_transform_and_filter_less_than.graphql.ron

```diff
@@ -0,0 +1,18 @@
+TestGraphQLQuery (
+    schema_name: "numbers",
+    query: r#"
+{
+    Number(min: 30, max: 30) {
+        ... on Composite {
+            value @output
+
+            primeFactor @fold @transform(op: "count") @filter(op: "<", value: ["$min_primes"]) {
+                prime: value @output
+            }
+        }
+    }
+}"#,
+    arguments: {
+        "min_primes": Uint64(2)
+    },
+)
```

Comment:
Not sure what this test case is testing? Also it seems to have no results because the `< 2` predicate on the fold filters everything out.

Reply:
I made the same test case for > and <, I didn't think it would be a problem to add another behavioral test case to ensure no breaks in existing behavior.